docs/gen-ai/gen-ai-agent-spans.md

```diff
@@ -0,0 +1,149 @@
+<!--- Hugo front matter used to generate the website version of this page:
+linkTitle: Generative AI traces
+--->
+
+# Semantic Conventions for GenAI agent and framework spans
+
+**Status**: [Experimental][DocumentStatus]
+
+<!-- Re-generate TOC with `markdown-toc --no-first-h1 -i` -->
+
+<!-- toc -->
+
+- [Spans](#spans)
+  - [Create Agent Span](#create-agent-span)
+- [Agent Execute Tool Span](#agent-execute-tool-span)
+
+<!-- tocstop -->
+
+Generative AI models can be trained to use tools to access real-time information or suggest a real-world action. For example, a model can leverage a database retrieval tool to access specific information, like a customer's purchase history, so it can generate tailored shopping recommendations. Alternatively, based on a user's query, a model can make various API calls to send an email response to a colleague or complete a financial transaction on your behalf. To do so, the model must not only have access to a set of external tools, it needs the ability to plan and execute any task in a self-directed fashion. This combination of reasoning, logic, and access to external information that are all connected to a Generative AI model invokes the concept of an agent.
+
+This document defines semantic conventions for GenAI agent calls that are defined by this [whitepaper](https://www.kaggle.com/whitepaper-agents).
+
+It MAY be applicable to agent operations that are performed by the GenAI framework locally.
+
+The semantic conventions for GenAI agents extend and override the semantic conventions for [Gen AI Spans](gen-ai-spans.md).
+
+## Spans
+
+### Create Agent Span
+
+Describes GenAI agent creation and is usually applicable when working with remote agent
```

Comment:
This idea of "creating an agent" is completely contradictory to any model I've seen in any agent framework.

Please could you add an explicit note saying that in the case of agent's defined in local code, this shouldn't be used.

I think this will be used very little.

Reply:
ptal at this comment https://github.com/open-telemetry/semantic-conventions/pull/1739/files#r1931341754